libx52/x52_control.c

```diff
@@ -168,7 +168,7 @@ int libx52_update(libx52_device *x52)
     unsigned int i;
     uint32_t update_mask;
     uint16_t value;
-    int rc;
+    int rc = LIBUSB_ERROR_OTHER;
```

Comment:
I think this should be initialized to 0. If the update mask is 0, then yes, rc would never get initialized, but we don't want to throw a failure.